projects/ngx-sweetalert2/src/lib/sweet-alert2.module.ts

```
@@ -0,0 +1,54 @@
+                    ? [SweetAlert2LoaderService, { provide: swalProviderToken, useValue: options.provideSwal }]
+                    : []),
+                ...(options.fireOnInit !== undefined
+                    ? [{ provide: fireOnInitToken, useValue: options.fireOnInit }]
+                    : []),
+                ...(options.dismissOnDestroy !== undefined
+                    ? [{ provide: dismissOnDestroyToken, useValue: options.dismissOnDestroy }]
+                    : []),
+            ],
+        };
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)